docs/bigcommerce-for-wordpress/extending-the-plugin/customization-guide.md

```diff
@@ -80,11 +91,11 @@ If you have a BigCommerce product ID, you can get the Product object using that
 
 `$product = \BigCommerce\Post_Types\Product\Product::by_product_id( $product_id );`
 
-In the context of many templates, the `$product` variable is already available to you. Check the docblock at the top of the template file to see which variables are defined in that scope. 
+In the context of many templates, the `$product` variable is already available to you. Check the docblock at the top of the template file to see which variables are defined in that scope.

```

Comment:
In the context of many templates, the `$product` variable is already available to you. Check the docblock at the top of the template file to see defined variables in that scope.